assets/app/features/upgrade/i18n/en.json

```diff
@@ -61,19 +61,19 @@
             },
             "database-configuration": {
                 "name": "",
-                "title": "Enter Credentials and/or Information to Create or Connect to an OpenStack PostgreSQL Database for the OpenStack Database Migration.",
+                "title": "Enter Credentials and/or Information to Create or Connect to an OpenStack PostgreSQL Server for the OpenStack Database Migration.",
                 "navigation-tabs": {
-                    "create-database": "Create New Database",
-                    "connect-database": "Connect to an Existing Database"
+                    "create-server": "Create New Server",
```

Comment:
Yup!
It will run a chef-solo client to install and configure the postgres server on the administration server.

Still this will change in the future as @kwwii said that all text and so on needs to be refined, so I wouldnt give it much thougth rigth now.